examples/property_prediction/pretrain_gnns/chem/model.py

```diff
@@ -0,0 +1,33 @@
+import torch.nn as nn
```

Comment:
Can we directly augment the built-in `GINPredictor`? E.g. add a flag to allow skipping readout.

Reply:
I misunderstood the origin implementation. We should skip the top Linear layer (`self.predict`) instead. How about skipping it with `n_task=0` flag?

Reply:
Deleted model.py. use built-in `GIN` and `GINPredictor` instead.